index.html

```diff
@@ -164,7 +164,7 @@ <h2 class="work_title">Tonic</h2>
             <ul class="work_cats">
               <li>Html</li>
               <li>Css</li>
-              <li>Javascript</li>
+              <li>Javascript</li> 
             </ul>
             <button type="button" class="default_btn">See More</button>
```

Comment:
- [ ] Its nice making your porfolio accessible, please make sure that the color contrast ratio passes. you can increase the color of your buttons while hovering.
### Color contrast ratio👇
![hover](https://user-images.githubusercontent.com/66992901/200554617-cb7a1571-efd6-4890-b753-41831f36e500.PNG)
![hover2](https://user-images.githubusercontent.com/66992901/200554706-a1836530-4893-4758-9f56-473dfd1818d7.PNG)

### Note: The page has a minimum contrast ratio of at least 4.5:1 for normal-size text.